cfg/cfg.go

```diff
@@ -66,6 +68,8 @@ func init() {
 	}
 	flag.StringVar(&AuthenticatedUserID, "authenticated-user-id", defaultAuthenticatedUserID, "OpenShift user's ID that should has access to API. Is used only if useBearerToken is configured")
 
+	flag.BoolVar(&UseTLS, "use-tls", false, "Serve content via TLS")
```

Comment:
If we use just boolean flag it makes sense to specify that `/var/serving-cert/tls.crt` and `/var/serving-cert/tls.key` are required if enabled.